### PR TITLE
EJB abbreviation removed

### PR DESCRIPTION
**Pull request by hendrikebbers**

### CODE REVIEW

File: spec/src/main/asciidoc/5_interceptor_ordering.adoc [hunk 1]
```diff
@@ -36,7 +36,7 @@ overridden.
 
 An extension specification may define
 alternative mechanisms (e.g., a deployment descriptor such as the CDI
-beans.xml [<<bib3>>] or the EJB _ejb-jar.xml_
+beans.xml [<<bib3>>] or the Enterprise Jakarta Beans _-jar.xml_
```
Comment (arjantijms) [inline spec/src/main/asciidoc/5_interceptor_ordering.adoc:39]: The file is still called _ejb-jar.xml_, which is one of the exceptions where we could continue using the term.

So it should then become Enterprise Jakarta Beans _ejb-jar.xml_

**Review by arjantijms — CHANGES_REQUESTED** [review]

File: spec/src/main/asciidoc/5_interceptor_ordering.adoc (hunk 1 above)
Comment (hendrikebbers) [reply spec/src/main/asciidoc/5_interceptor_ordering.adoc]: Done

**Comment by hendrikebbers** [comment]
@arjantijms sorry for all the broken PRs. Not now it looks good ;)

**Review by arjantijms — APPROVED** [review]
LGTM! thanks :)

**Comment by tkburroughs** [comment]
Note, there is no such thing as "Enterprise Jakarta Beans", it should be "Jakarta Enterprise Beans".

**Comment by arjantijms** [comment]
> Note, there is no such thing as "Enterprise Jakarta Beans", it should be "Jakarta Enterprise Beans".

You're absolutely right. Big shame on me. I somehow read completely over that and even used it myself, maybe because the traditional Java Enterprise Beans full term is still so fresh in my mind...

**Comment by arjantijms** [comment]
@tkburroughs I did a PR here to fix it: https://github.com/eclipse-ee4j/interceptor-api/pull/78

Sorry for the noise.

**Pull request merged**